packages/dapp/src/types/resourceKey.ts

```diff
@@ -0,0 +1,64 @@
+import { ResourceKey as I18nResourceKey } from 'i18next';
+
+interface ExtendedResourceKey {
```

Comment:
Wow, I don't think we need these types 🙂 Let's discuss.

Reply:
Yes, seeing it now it feels like this is just a pain in the a** to keep that manually up to date and should be removed.